api/WakeLockSentinel.json

```diff
@@ -14,7 +14,7 @@
           "chrome_android": "mirror",
           "edge": "mirror",
           "firefox": {
-            "version_added": "preview"
+            "version_added": "126"
```

Comment:
All changes in this file match the corresponding Bugzilla issue. Related docs work tracking in https://github.com/mdn/content/issues/33047